src/github/pull-request.ts

```diff
@@ -25,12 +26,11 @@ export const pullRequestWebhookHandler = async (context: CustomContext, jiraClie
 	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, jiraClient.baseURL, context.log);
 
 	context.log = context.log.child({
-		jiraHostName: jiraClient.baseURL,
+		jiraHostName: createHashWithSharedSecret(jiraClient.baseURL),
 		installationId: githubInstallationId,
-		orgName: owner,
+		orgName: createHashWithSharedSecret(owner),
 		pullRequestNumber,
-		pullRequestId,
-		payload: context.payload
```

Comment:
Completely removing the payload here, seems a bit silly to hash and we can't guarantee the contents is not UGC/PII so leaning towards safer.